File: FootScout-Vue.WebAPI/Models/DTOs/UserDTO.cs
namespace FootScout_Vue.WebAPI.Models.DTOs
{
    public class UserDTO
    {
        public string Id { get; set; }
        public string Email { get; set; }
        public string PasswordHash { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Location { get; set; }
        public DateTime CreationDate { get; set; }
    }
}

File: FootScout-Vue.WebClient/src/models/dtos/UserDTO.ts
// Reprezentuje pełne dane użytkownika, w tym identyfikator, e-mail, hasło, imię, nazwisko, 
// telefon, lokalizację i datę utworzenia konta.

export interface UserDTO {
    id: string;
    email: string;
    passwordHash: string;
    firstName: string;
    lastName: string;
    phoneNumber: string;
    location: string;
    creationDate: string;
}